modules/rosa-sts-associating-your-aws-account.adoc

```diff
@@ -13,20 +13,38 @@ endif::[]
 ifeval::["{context}" == "rosa-quickstart"]
 :quickstart:
 endif::[]
+ifeval::["{context}" == "rosa-hcp-sts-creating-a-cluster-quickly"]
+:rosa-hcp:
+endif::[]
+ifeval::["{context}" == "rosa-sts-creating-a-cluster-quickly"]
+:rosa-standalone:
+endif::[]
```

Comment:
Along similar lines to Jeana's comment in https://github.com/openshift/openshift-docs/pull/55565#discussion_r1130039490, I'm wondering if these newly introduced `ifevals` need matching entries at the bottom of the file to unset the variables, too.

There's a few more below like this I'll highlight.